server/protocols/jmap/src/main/java/org/apache/james/jmap/DownloadServlet.java

```diff
@@ -118,6 +118,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
             Attachment attachment = attachmentManager.getAttachment(AttachmentId.from(blobId), mailboxSession);
             IOUtils.copy(attachment.getStream(), resp.getOutputStream());
 
+            resp.setContentLength((int) attachment.getSize());
```

Comment:
That's a bug !
I can work around it by using :  
`response.setHeader("Content-Length", String.valueOf(attachment.getSize());`